classes/item.rb

```diff
@@ -0,0 +1,38 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :archived
+  attr_reader :author, :label, :genre
+
+  def initialize(publish_date, archived: false, id: Random.rand(1...1000))
+    @id = id
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def genre=(genre)
+    @genre = genre
+    genre.items << self unless genre.items.include?(self)
+  end
+
+  def author=(author)
+    @author = author
+    author.items << self unless author.items.include?(self)
+  end
+
+  def label=(label)
+    @label = label
+    label.items << self unless label.items.include?(self)
+  end
+
+  def move_to_archive
```

Comment:
- [ ] Kindly write tests for all the attributes and public methods of the `Item` class. Remember that you ought to write tests for all classes in this project. Kindly implement this. 👍 😎